### DISCUSSION TITLE
Show account switcher when page doesn’t load due to permissions

### DISCUSSION THREAD

**Discussion by sarahhodne** [community/community#132647] [Mobile]
I'm signed in to multiple accounts in the app, but when I tap a link in a different app that the currently selected account doesn't have permissions to view, then I just get the error page with no ability to switch accounts, I need to back out, switch accounts, then switch back to the first app and tap the link again.

It would be nice if the account switcher that shows for pages that do load also show here, or if there was some way to quickly switch accounts in this case and reload the page without having to go back and find the link again.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.169.0 (136088047)
Device: iPhone 14 Pro Max (iOS 17.5.1)
</details>

**Comment by stevepopovich** [comment]
Hi @sarahhodne thanks for writing in. We expect this switcher to show in this case! So if it isn't showing, that's a bug. What deeplink are you using where you don't see the switcher?

**Reply by sarahhodne** [reply]
Looking more closely it looks like it does show up for a brief moment before the page actually loads, but then I get the "Something went wrong." page and the switcher disappears right when that shows up.

The link I clicked is on the form `https://github.com/[org]/[repo]/pull/[reponumber]#issuecomment-[commentnumber]` (from Slack, if it matters) for a private repo that I don't have access on the active account.

**Reply by stevepopovich** [reply]
Gotcha. Interesting! After some testing I can totally reproduce! We should absolutely show the switcher here. I will file a bug internally.

As a "workaround", you should see a pop up towards the bottom of the screen that says "Switch Account". Tapping on that should work. Just gotta be fast enough 😄
